### ISSUE TITLE
[feature] create /health endpoint

### ISSUE THREAD

**Issue by Woitekku** [hoppscotch/proxyscotch#92]
It would be good to have a healthcheck endpoint like /health that will return 200 if proxy is up and running.
Very useful for kubernetes envs.

For now I'm doing / and accept 301 as "healthy" but it's not fully correct/ valid.

**Comment by monwolf** [comment]
Yeah, it would be fine. It would also be fine to expose some kind of basic metric for monitoring the usage.
